src/game_main.cpp

```diff
@@ -50,36 +48,11 @@ namespace rigel {
 
 using namespace engine;
 using namespace sdl_utils;
+using namespace std::chrono;
 
 using RenderTargetBinder = renderer::RenderTargetTexture::Binder;
 
-namespace {
-
-template <typename Callback>
-class CallOnDestruction {
```

Comment:
In https://github.com/lethal-guitar/RigelEngine/pull/561, I've moved this to a dedicated new file